src/components/Admin/__snapshots__/Admin.test.jsx.snap

```diff
@@ -284,7 +293,16 @@ exports[`<Admin /> renders correctly with dashboard analytics data renders # cou
                   </div>
                   <div>
                     <span
+<<<<<<< HEAD
+                      aria-hidden={true}
+                      className="fa fa-caret-down"
+                      id="Icon30"
+                    />
+                    <span
+                      className="sr-only"
+=======
                       className="pgn__icon"
+>>>>>>> 2d7e691905069d068295f6743f7aa736c671f40e
```

Comment:
Hi @brobro10000, You have pushed some conflicted code in the recent commit. This PR is currently in `Draft` state, once the [react upgrade](https://github.com/openedx/frontend-app-admin-portal/pull/1009) gets merged, I will rebase this PR.